### PR TITLE
feat: Add Hashstack

### PR DESCRIPTION
**Pull request by web3Gurung**
Adding Hashstack to the existing list of StarkNet based Dapps on dappland. Please let me know if something has to be fixed. The json file and collaterals are according to the [requirements](https://github.com/argentlabs/dappland#-add-your-dapp-to-dappland).

Thank you!

### CODE REVIEW

File: data/hashstack.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "description": "Hashstack provides a permissionless under-collateralised lending solution for crypto-native retail, enabling them to borrow up to 300% of the collateral. Product status: Public testnet on Starknet.",
+  "short_description": "Permissionless undercollateralized loans",
+  "tags": [
+    "DeFi"
+  ], 
+  "contracts": [],
+  "goerliContracts": [
+    {
+      "name": "Diamond Contract",
+      "address": "https://testnet-2.starkscan.co/contract/0x05ad65a92e83d652175aa914e922afd2917f6a9520047eaa0da0684749163fc4"
```
Comment (graemeblackwood) [inline data/hashstack.json:12]: @web3Gurung please update this to the contract addresses only

**Review by graemeblackwood — CHANGES_REQUESTED** [review]
The icon will be cropped to a circle using CSS so I recommend it be square, otherwise we may have a pixelated outline if the crops don't match exactly (this is common when using CSS border-radius around a circled image.

Other than this, we just need the contract address fixed and I'll merge.

Thanks!

File: data/hashstack.json (hunk 1 above)
Comment (web3Gurung) [reply data/hashstack.json:12]: I will make the required changes, thank you :)

**Comment by web3Gurung** [comment]
Hey @graemeblackwood, I've updated the logo to square shape and also updated the contract address. Please have a look :)

**Pull request merged**